### DISCUSSION TITLE
using a few the same modules on one page

### DISCUSSION THREAD

**Discussion by ijcnvv** [vuejs/pinia#1034] [Help and Questions] [answered]
is it possible?

in my case I use store module for comments, so I register it (in Vuex) by dynamic key `this.$store.registerModule(anyDynamicKey, store)` and then I use it for any features like news etc.

**Comment by posva** [comment, marked as answer]
Instead, store a Map inside the store state and pass the key to each action manipulating comments

**Comment by ijcnvv** [comment]
sure, but maybe another way exists there
